scout/load/phenotype.py

```diff
@@ -208,10 +210,9 @@ def load_disease_terms(
                 disease_id="OMIM:{0}".format(disease_number),
                 hpo_term_to_symbol=hpo_term_to_symbol,
             )
+            disease_objs.append(build_disease_term(disease_info, genes))
 
-            disease_obj = build_disease_term(disease_info, genes)
-
-            adapter.load_disease_term(disease_obj)
+    adapter.load_disease_terms(disease_objs)
```

Comment:
We have the RAM nowadays. This should be faster as well, in addition to allow testing. Wouldn't about here be a good point to drop the old as well?

Reply:
It's super fast, not so many items to load. Good point, we can drop the old terms here, so we also avoid to have the check function before, I think?

Reply:
If you mean the file check function I guess it doesn't matter as much when there is validation prior to dropping, but it sure doesn't hurt just to check if the files are there before doing anything. 😊

Reply:
I'll reintroduce that code. It would have crashed anyway 😆